### PR TITLE
Handle empty IsMaxItemsOne arrays in Diff.

### PR DESCRIPTION
**Pull request by pgavlin**
This code was not updated to match the behavior in MakeTerraformInput.

Fixes https://github.com/pulumi/pulumi-aws/issues/821.

### CODE REVIEW

**Comment by lukehoban** [comment]
Just for reference - this is follow up to changes made in https://github.com/pulumi/pulumi-terraform-bridge/pull/81?

**Comment by pgavlin** [comment]
> Just for reference - this is follow up to changes made in #81?

Yes.

File: pkg/tfbridge/diff.go [hunk 1]
```diff
@@ -70,7 +70,11 @@ func visitPropertyValue(name, path string, v resource.PropertyValue, tfs *schema
 	rawNames bool, visitor propertyVisitor) {
 
 	if IsMaxItemsOne(tfs, ps) {
-		v = resource.NewArrayProperty([]resource.PropertyValue{v})
+		if v.IsNull() {
+			v = resource.NewArrayProperty([]resource.PropertyValue{})
+		} else {
+			v = resource.NewArrayProperty([]resource.PropertyValue{v})
+		}
```
Comment (lukehoban) [inline pkg/tfbridge/diff.go]: Can we cover this with test coverage?

Comment (pgavlin) [reply pkg/tfbridge/diff.go]: Yup, I'll go ahead and do that before merging

**Pull request merged**